apps/api/src/common/hotcache.ts

```diff
@@ -0,0 +1,22 @@
+class CacheObject {
+    constructor(public value: any, public expiration: number) {}
+    isExpired() {
+        return this.expiration < Date.now()
+    }
+}
+
+export class HotCache {
+    private cache: { [key: string]: CacheObject } = {}
+
+    get(key: string) {
```

Comment:
`get` and `set` are reserved words. Not that it is an issue, but consider renaming those methods, to avoid any possible confusion.

Reply:
done.